frappe/__init__.py

```diff
@@ -155,6 +155,8 @@ def set_user_lang(user: str, user_language: str | None = None) -> None:
 debug_log = local("debug_log")
 message_log = local("message_log")
 
+force_debug_logs = local("force_debug_logs")
```

Comment:
`frappe.log(..., force=True)` will be better?

This way we can ignore logging param in site config?